src/neo/Network/P2P/Payloads/BlockBase.cs

```diff
@@ -118,7 +119,7 @@ public virtual bool Verify(StoreView snapshot)
             if (prev_header == null) return false;
             if (prev_header.Index + 1 != Index) return false;
             if (prev_header.Timestamp >= Timestamp) return false;
-            if (!this.VerifyWitnesses(snapshot, 1_00000000)) return false;
+            if (!this.VerifyWitnesses(snapshot, 3000000 * NativeContract.Policy.GetBaseExecFee(snapshot))) return false;
```

Comment:
I think this can be fixed. Because as the CPU speed increases, I will lower the price, but at the same time we will allow more complex verification scripts.

Reply:
Reverted this part of the change.

Though I think `ECDsaVerifyPrice` can be expressed in NOPs too because it clearly is CPU-bound and this ratio is expected be more stable from CPU to CPU than execution time. And then verification limits for blocks or consensus payloads are dominated by `ECDsaVerifyPrice`.